app/models/follow_up.rb

```diff
@@ -2,34 +2,24 @@
 
 require 'rest-client'
 
-class FollowUp
-  include ActiveModel::Validations
-
-  attr_accessor :email, :name, :language, :destination
+class FollowUp < ActiveRecord::Base
+  belongs_to :language
+  belongs_to :destination
 
   validates :email, presence: true, email_format: { message: 'Invalid email address' }
   validates :language, presence: true
   validates :destination, presence: true
 
-  def initialize(email, language_id, destination_id, name = nil)
-    self.email = email
-    self.language = Language.find(language_id)
-    self.destination = Destination.find(destination_id)
-    self.name = name
-  end
-
   def send_to_api
-    validate_fields
+    save!
+
+    Rails.logger.info "Sending follow up with id: #{id}."
```

Comment:
How often will this get called? Just want to be careful not to log to `info` with anything that's too verbose.

Reply:
It will be logged every time a follow up is sent

Reply:
I get that, I just don't have any sense of how often that will be. Is this something you plan to search for in the logs later? What's the value in logging this? Presumably it's getting logged to the database anyway?

Reply:
I want to log the ID so I can look it up later if there's a failure